### ISSUE TITLE
Crash on Client

### ISSUE THREAD

**Issue by kenijey** [Nuchaz/BiblioCraft#469]
[crash-2017-06-05_21.16.51-client.txt](https://github.com/Nuchaz/BiblioCraft/files/1052827/crash-2017-06-05_21.16.51-client.txt)

**Comment by gchpaco** [comment]
I have a similar crash.  Looks related to #404.

**Comment by Nuchaz** [comment]
I've got a fix in the works, I was just finally able to narrow down what is crashing by downloading someone elses world and modpack and was reliably crashing. I am going to try to have a fix ready as soon, hopefully by this weekend at the latest.

**Comment by kreezxil** [comment]
I'm so glad to hear you say these words. Thank you thank you. If you have a jenkins you want me to try out I can do that. I have the capability to create any test configuration at will.

**Comment by Nuchaz** [comment]
I have a fix out for this. I rolled the fix into BiblioCraft v2.1.4 for MC 1.9.4, BiblioCraft v2.2.4 for MC 1.10.2 and BiblioCraft v2.3.2 for MC 1.11.2. Closing the issue. Let me know if you experience anything weird.

**Comment by kreezxil** [comment]
Will do.

On Jun 7, 2017 7:36 PM, "Nuchaz" <notifications@github.com> wrote:

I have a fix out for this. I rolled the fix into BiblioCraft v2.1.4 for MC
1.9.4, BiblioCraft v2.2.4 for MC 1.10.2 and BiblioCraft v2.3.2 for MC
1.11.2. Closing the issue. Let me know if you experience anything weird.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/Nuchaz/BiblioCraft/issues/469#issuecomment-306964695>,
or mute the thread
